package.json

```diff
@@ -42,6 +42,9 @@
     "prod": "run-s update-oss-docs build",
     "checklinks": "yarn dlx hyperlink --canonicalroot https://www.oddbird.net/ -r -p --skip \"_site/feed.atom\" --skip \"docs.djangoproject.com\" --skip \"/docs/\" --skip \"codepen.io\" --skip \"github.com\" --skip \"caniuse.com\" --skip \"youtu.be\" --skip \"stackoverflow.com/users/\" _site/index.html"
   },
+  "dependencies": {
+    "@algolia/algoliasearch-netlify-frontend": "^1.0.14"
+  },
```

Comment:
@jgerigmeyer I had a merge conflict here so I accepted the latest on main and then added in the dependencies with only the search package... wasn't sure if you wanted to move accoutrement, ccs and css remedy here like it was in this branch before this commit or move search to the devDependencies list?